cli/src/lib.rs

```diff
@@ -67,14 +67,12 @@ impl Default for Arguments {
             submit_genesis: SUBMIT_GENESIS,
             genesis_path: Some(ConfigPath::default(GENESIS_PATH).unwrap_or_else(|e| {
                 panic!(
-                    "Default genesis path `{}` has extension, but it should not have one. {e:?}",
-                    GENESIS_PATH
+                    "Default genesis path `{GENESIS_PATH}` has extension, but it should not have one. {e:?}",
                 )
             })),
             config_path: ConfigPath::default(CONFIGURATION_PATH).unwrap_or_else(|e| {
                 panic!(
-                    "Default config path `{}` has extension, but it should not have one. {e:?}",
-                    GENESIS_PATH
+                    "Default config path `{GENESIS_PATH}` has extension, but it should not have one. {e:?}",
```

Comment:
@Arjentix I believe you've made a change recently in a case like this where you were able to make it a `lazy_static`?

Reply:
Shows up due to conflict. Will disappear after rebase.